### PR TITLE
HBW-338 Add locale to static field

### PR DESCRIPTION
**Pull request by sergeichukd**

### CODE REVIEW

File: config/locales/bp/ru.new_customer.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ ru:
       homsOrderDataCustomerStreet: Улица
       homsOrderDataCustomerHouse: Дом
       homsOrderDataCustomerEntrance: Подъезд
+      staticField: <div class="alert alert-danger">Сообщение из локали на русском языке</div>
```
Comment (TimAle) [inline config/locales/bp/ru.new_customer.yml:10]: There is no such field on this form in the demo process.

File: hbw/app/javascript/packs/hbw/components/form/static.js.jsx [hunk 2]
```diff
@@ -1,16 +1,23 @@
 import compose from 'shared/utils/compose';
 import cx from 'classnames';
 import { withConditions, withErrorBoundary } from 'shared/hoc';
+import TranslationContext from 'shared/context/translation';
+import { useContext } from 'react';
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/form/static.js.jsx:5]: To line 1.

**Review by TimAle — CHANGES_REQUESTED** [review]

File: config/locales/bp/ru.new_customer.yml [hunk 3]
```diff
@@ -7,6 +7,7 @@ ru:
       homsOrderDataCustomerStreet: Улица
       homsOrderDataCustomerHouse: Дом
       homsOrderDataCustomerEntrance: Подъезд
+      static: <div class="alert alert-success"><h4>Подключение доступно</h4> <small>Решение основано на автоматической проверке доступа в биллинге</small></div>
```
Comment (TimAle) [inline config/locales/bp/ru.new_customer.yml:10]: This form has no such field, check actual business process, please
https://github.com/latera/camunda-ext/blob/b0647bc3ad79a3bfcc06ecb00c388f8b2eb95ef9/demo_processes/new_customer/src/main/resources/forms/plan.yml#L4

**Review by TimAle — CHANGES_REQUESTED** [review]

File: config/locales/bp/ru.new_customer.yml (hunk 3 above)
Comment (sergeichukd) [reply config/locales/bp/ru.new_customer.yml]: done

**Review by 0xf10yd — APPROVED** [review]

File: hbw/config/yml_api.test.camunda.yml [hunk 4]
```diff
@@ -1863,21 +1863,80 @@ get:
...
+                type: number
+                alignment: right
+                precision: 2
+            choices:
+              - [1, Home alone,                 30,  0,   5]
+              - [2, Movie lover,                50,  101, 6.5]
+              - [3, Game lover,                 100, 0,   7]
+              - [4, Like a pro,                 200, 0,   8]
+              - [5, For a family,               50,  201, 7.5]
+              - [6, For a family with teenager, 200, 201, 10]
```
Comment (TimAle) [inline hbw/config/yml_api.test.camunda.yml:1901]: You don't need all this data, do you?

File: hbw/config/yml_api.test.camunda.yml [hunk 5]
```diff
@@ -1863,21 +1863,80 @@ get:
               - Nizhniy Novgorod
               - Samara
```
Comment (TimAle) [inline hbw/config/yml_api.test.camunda.yml:1864]: And this ones.

**Review by TimAle — CHANGES_REQUESTED** [review]

File: hbw/config/yml_api.test.camunda.yml (hunk 4 above)
Comment (sergeichukd) [reply hbw/config/yml_api.test.camunda.yml:1901]: Yes I do

File: hbw/config/yml_api.test.camunda.yml (hunk 5 above)
Comment (sergeichukd) [reply hbw/config/yml_api.test.camunda.yml]: done

**Review by TimAle — APPROVED** [review]

**Pull request merged**